docs/process/rust_versions.md

```diff
@@ -3,7 +3,7 @@
 
 ## For ICU4X users
 
-Each ICU4X release and its patches guarantee a minimum supported rust version (MSRV) of four Rust versions before the release, and attempt an MSRV of six Rust versions. For example, if today's current Rust stable version is 1.70, and we release ICU4X 1.3, then we guarantee that ICU4X 1.3 will have an MSRV of 1.66 or earlier. We will *attempt* an MSRV of 1.64.
+Each ICU4X release guarantee a minimum supported rust version (MSRV) of four Rust versions before the release, and attempt an MSRV of six Rust versions. For example, if today's current Rust stable version is 1.70, and we release ICU4X x.y.0, then we guarantee that ICU4X x.y.z will have an MSRV of 1.66 or earlier. We will *attempt* an MSRV of 1.64.
```

Comment:
I find the previous wording of "release and its patches" clearer.

Reply:
Same. I'm fine with including the x.y stuff though

Reply:
Reverted